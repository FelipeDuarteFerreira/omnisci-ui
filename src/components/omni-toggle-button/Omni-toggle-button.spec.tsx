import React from "react"
import { render } from "enzyme"

import OmniToggleButton from "./Omni-toggle-button"

test("Should render a toggle", () => {
  const props = {
    label: "Label"
  }
  const component = render(<OmniToggleButton {...props} />)

  expect(component).toMatchSnapshot()
})
