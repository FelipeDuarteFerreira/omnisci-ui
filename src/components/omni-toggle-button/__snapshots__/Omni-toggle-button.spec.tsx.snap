// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a toggle 1`] = `
<div
  class="omni-toggle-button"
>
  <button
    class="secondary-state secondary-state mdc-button mdc-button--unelevated"
  >
    <span
      class="mdc-button__label"
    />
  </button>
</div>
`;
